cmd/nerdctl/logs.go

```diff
@@ -75,32 +75,33 @@ func logsAction(cmd *cobra.Command, args []string) error {
 	}
 	defer cancel()
 
+	follow, err := cmd.Flags().GetBool("follow")
+	if err != nil {
+		return err
+	}
+	tail, err := cmd.Flags().GetString("tail")
```

Comment:
maybe we need to add some docs and limit code here because the journals and fluentd driver are not support this new argument yet

Reply:
Should be an another PR, not related to this PR

Reply:
My bad, I will open another PR